### ISSUE TITLE
test_curim  fails

### ISSUE THREAD

**Issue by staier** [mrmetech/Curium#2]
despite the   broken  build process i  managed to  get  the thing  compiled 
,  however  test_curim  fails  :

test_curium test_curium: chainparams.cpp:142: CMainParams::CMainParams(): Assertion `consensus.hashGenesisBlock == uint256S("0x00000e784f88ffd07483e5e2786e002a780d35dfd4142c8e70c43fb7bb2af937")' failed. Aborted (core dumped)

**Comment by mrmetech** [comment]
Use the release not the beta

**Comment by staier** [comment]
Thanks .
However  what  is the  release? 
your  site is down ,   two  last  tags  contain  no autogen.sh nor  makefile.
So how exactly  can  i  obtain  binary ?

**Comment by mrmetech** [comment]
On github there is a release tab or you can select tags we are working on
upgrading our wallet rn and give a 1 btc bounty for someone to do it and a
long term job as am very busy atm.

**Comment by staier** [comment]
ok ,  last  question.
am  i  correct  you  only  have windows binary  in  the  release ?

**Comment by mrmetech** [comment]
Yes but you can compile the linux one easily we will be releasing more
after the update if you could spread this around Hi am from a new Project
Called Curium and would like to hire someone to help us upgrade from a
version of dash 10 to dash 12. We are a newer project and need the upgrade.
For that we offer 1 BTC and a long term job. Contact me on discord at
mrmetech for more info and https://discord.gg/Ye36CDh and check out are
Bitcoin talk forum. At https://bitcointalk.org/index.php?topic=2754529.0

**Comment by jbgraindorge** [comment]
can not build neither

>libtoolize: copying file 'build-aux/m4/lt~obsolete.m4'
configure.ac:10: installing 'build-aux/compile'
configure.ac:5: installing 'build-aux/config.guess'
configure.ac:5: installing 'build-aux/config.sub'
configure.ac:9: installing 'build-aux/install-sh'
configure.ac:9: installing 'build-aux/missing'
Makefile.am:78: error: CPPFLAGS_FOR_BUILD must be set with '=' before using '+='
Makefile.am:79: error: CFLAGS_FOR_BUILD must be set with '=' before using '+='
Makefile.am: installing 'build-aux/depcomp'
parallel-tests: installing 'build-aux/test-driver'
autoreconf: automake failed with exit status: 1

**Comment by mrmetech** [comment]
Use release that is a beta build

**Comment by staier** [comment]
tags  do  not compile for unix either

**Comment by mrmetech** [comment]
Ok but that is why there is a release tab

**Comment by staier** [comment]
sure ,  but  who  does blockchain  development   for windows anyway?

**Comment by jbgraindorge** [comment]
I was able to compile 2 days ago, from the makefile.unix in src , now it's gone, and can not build with the autogen.sh way..., the zip src is release 0.10, same as I have, I want source, not binary

**Comment by mrmetech** [comment]
The source is under release in a zip
